kubeflow/core/prototypes/cloud-endpoints.jsonnet

```diff
@@ -0,0 +1,25 @@
+// @apiVersion 0.1
+// @name io.ksonnet.pkg.cloud-endpoints
+// @description Provides cloud-endpoints prototypes for creating Cloud Endpoints services and DNS records.
+// @shortDescription Cloud Endpoint domain creation.
+// @param name string Name for the component
+// @param secretName string Name of secret containing the json service account key.
+// @optionalParam secretKey string cloudep-sa.json Name of the key in the secret containing the JSON service account key.
+// @optionalParam namespace string null Namespace to use for the components. It is automatically inherited from the environment if not set.
+
+// TODO(https://github.com/ksonnet/ksonnet/issues/222): We have to add namespace as an explicit parameter
```

Comment:
Delete this since the issue has been resolved you are inheriting namespace below.